.github/pull_request_template.md

```diff
@@ -1,18 +1,18 @@
-You are about to submit a new Pull Request. Before continuing make sure you read the [contributing guidelines](CONTRIBUTING.md) and you comply with the following criteria:
+You are about to submit a new Pull Request. Before continuing make sure you read the [contributing guidelines](CONTRIBUTING.md) and that you comply with the following criteria:
 
-- [ ] You have stick to Python. Talk with us before adding other programming languages to HADDOCK3
+- [ ] You have sticked to Python, talk with us before adding other programming languages to HADDOCK3
```

Comment:
The comma should be a dot, i.e.g You have sticked to Python. Please talk to us before...